### DISCUSSION TITLE
Import points from Trekarta Lite. Импорт точек из Трекарта Лайт

### DISCUSSION THREAD

**Discussion by Whitebird2** [andreynovikov/trekarta#218] [Q&A]
Hi All, I had a lot of personal navigation points stored in Trekarta Lite. Unfortunately since some time the Trekarta Lite doesn't open on smartphone. I installed Trekarta. It opens well, but I don't see my old navigation points. How to import those?

Привет всем. У меня на смартфоне установлена программа Трекарта Лайт. Там я сохранял точки из своих походов. С некоторого времени программа перестала открываться. Я установил последнюю версию Трекарта, но там моих точек нет и я не понимаю, как импортировать старые. Пожалуйста подскажите.

**Comment by andreynovikov** [comment]
Необходимо подключиться к телефону с помощью компьютера или установить на телефон какой-нибудь хороший менеджер файлов, я использую Sold Explorer. Найти папку `/Android/data/mobi.maptrek.lite/files/databases`, в ней найти файл `waypoints.sqlitedb`. В этом файле хранятся все точки. Его нужно сохранить в надёжное место, чтобы не потерять. После этого его нужно скопировать в папку `/Android/data/mobi.maptrek/files/databases`. Там уже будет такой-же файл, но если вы не создавали точки в Трекарте, то его можно безболезненно перезаписать.

**Comment by Whitebird2** [comment]
Спасибо, все получилось! Только почему-то старый файл базы данных был расположен в другом месте - в корне внутренней памяти смартфона папка Trekarta/databases

**Comment by andreynovikov** [comment]
Это значит какая-то старая версия была.
